apps/api/tests/test_summary_handler.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import AsyncMock, patch
+
+from admin.summary_handler import applicant_summary
+
+
+@patch("services.mongodb_handler.retrieve", autospec=True)
+async def test_applicant_summary(mock_mongodb_handler_retrieve: AsyncMock) -> None:
+    """Test applicant summary counts by status."""
+    mock_mongodb_handler_retrieve.return_value = (
+        [{"status": "ACCEPTED"}, {"status": "REJECTED"}] * 20
+        + [{"status": "CONFIRMED"}] * 24
+        + [{"status": "WAITLISTED"}, {"status": "WAIVER_SIGNED"}] * 3
```

Comment:
Suggestion: For the sake of coverage, do we also want to include counts for the other statuses?